packages/vuetify/src/components/VFileInput/VFileInput.ts

```diff
@@ -141,6 +141,18 @@ export default VTextField.extend({
       },
       immediate: true,
     },
+    value: {
```

Comment:
You don't need object-style watch here.

Reply:
You should add a linting rule for this... Otherwise, how are we supposed to know all the "implicit" rules?

Reply:
okay, we'll consider it

Reply:
but for now please fix it, it's not required here.

Reply:
But why? I've never seen anyone use a handler for a watcher that wasn't using immediate or deep. It's not an implicit rule and more of a general code style.

Reply:
Upvote to @johnleider, object style must only be used in deep/immediate

Reply:
I agree the style is better. It actually is written like that because I was testing with `immediate: true` and forgot to cleanup styling afterwards. An eslint rule would have avoided this conversation and make us all gain some time ;-)